src/gui/AstroCalcDialog.cpp

```diff
@@ -3018,11 +3018,15 @@ pSEparameter partialSolarEclipse() {
 	double L1 = bessel.L1;
 	double L2 = bessel.L2;
 	const double mu = bessel.mu;
-	constexpr double e2 = 0.00669398;
-	// e^2 = 0.00669398 : Earth flattening parameter
-	// f = 298.25642 : e^2 = 2f-f^2
-	// Source: IERS Conventions (2003)
+	constexpr double e2 = 0.00669438;
+	// e^2 = 0.00669438 : Earth flattening parameter
+	// Inverse flattening 1/f = 298.257223563 : e^2 = 2f-f^2
```

Comment:
We are actually using this as property of the Planet object Earth. See ssystem_major.ini entry for Earth: 
`oblateness=0.003352810664747481
`
This is f. 
The parameter is used in Planet::getOneMinusOblateness() and therefore in Planet::getRectangularCoordinates(), which was crucial to get ... accurate eclipse simulation for 2017. 

For consistency I would prefer if you don't hardcode the value here but use Planet::getOneMinusOblateness(), should we ever want to change to the IERS2003 or other values. But please keep the documentation, this is important for traceability.

Reply:
Thanks, I'll add it in my to do list.